spec/api-session-spec.ts

```diff
@@ -1451,6 +1451,7 @@ describe('session module', () => {
       w.webContents.executeJavaScript(`
         var iframe = document.createElement('iframe');
         iframe.src = '${loadUrl}';
+        iframe.allow = 'clipboard-read';
```

Comment:
https://www.w3.org/TR/permissions-policy-1/ is now enabled for cross origin frames, should be documented under breaking changes.